bots/common/commands_dict.py

```diff
@@ -544,23 +543,23 @@ def get_tickers() -> List[str]:
         "required": {"ticker": tickers, "interval": opt_intervals},
         "optional": {"past_days": re_int, "start": re_date, "end": re_date},
     },
-    "opt-overview": {
+    "opt_overview": {
         "function": overview_opt_command,
         "required": {"ticker": tickers, "expiry": re_date},
```

Comment:
I was able to get the expiry dates args to work only after I changed `re_date` to `re_name`. Issue with the re.Pattern?